core/trino-main/src/main/java/io/trino/execution/QueryManagerConfig.java

```diff
@@ -637,6 +638,19 @@ public boolean isRetryPolicyAllowed()
         return allowedRetryPolicies.contains(retryPolicy);
     }
 
+    public boolean isUseOfExchangeForQueryRetryPolicyAllowed()
+    {
+        return useOfExchangeForQueryRetryPolicyAllowed;
+    }
+
+    @Config("use-of-exchange-for-query-retry-policy-allowed")
```

Comment:
What’s the use case for this? Every option we add makes the system harder to configure and reason about.

Also, the name of that option is a mouthful. We need a better name.